extensions/tracing.md

```diff
@@ -18,7 +18,7 @@ With this instrumentation, we can chain synchronous and asynchronous commands an
 </dependency>
 ```
 
-The first dependency is [Spring Boot starter for Axon Tracing extension](../getting-started/maven-dependencies.md#axon-tracing-spring-boot-starter), which is the quickest start in to an extension configuration.
+The first dependency is [Spring Boot starter for Axon Tracing extension](https://github.com/domaincomponents/reference-guide/tree/375d9d62c18f30922cb3bc5be76351deeb9a64f0/getting-started/maven-dependencies.md#axon-tracing-spring-boot-starter), which is the quickest start in to an extension configuration.
```

Comment:
Why is the GitHub URL stuff needed for linking here?

Reply:
Resolved